### PR TITLE
Add update.json to /mousetrap

### PR DESCRIPTION
**Pull request by fatbattk**
For auto-updates.

### CODE REVIEW

File: files/mousetrap/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "mousetrap",
+  "repo": "ccampbell/mousetrap",
+  "files": {
+    "include": ["mousetrap.min.js", "mousetrap.js", "./plugins/**/*.js"],
```
Comment (jimaek) [inline files/mousetrap/update.json]: `"./plugins/**/*.min.js"` This will be better for production

**Comment by fatbattk** [comment]
@jimaek Agree and updated. But why is auto-check failing for this?

    Issues validating update.json for mousetrap: No JSON object could be decoded

**Comment by jimaek** [comment]
Not sure. Lets see what happens.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**